flake.nix

```diff
@@ -0,0 +1,95 @@
+{
+  description = "Traewelling";
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    devenv.url = "github:cachix/devenv";
+  };
+  outputs = inputs @ {flake-parts, ...}:
+    flake-parts.lib.mkFlake {inherit inputs;} {
+      imports = [
+        inputs.devenv.flakeModule
+      ];
+      systems = [
+        "x86_64-linux"
+        "aarch64-linux"
+        "x86_64-darwin"
+        "aarch64-darwin"
```

Comment:
Have you tested this flake on all these systems or is this just guess on which will probably work?

Reply:
I only tested on x86_64-linux, but I'm very sure it will work on all of the other platforms

Reply:
Tested with `aarch64-darwin` and working like a charm.

Reply:
aarch64-linux works as well :)
![träwelling running on an android phone](https://cdn.discordapp.com/attachments/778653325637320724/1146221766700707911/Screenshot_20230830-011538.png)